Documentation/gettingstarted/clustermesh.rst

```diff
@@ -68,12 +68,20 @@ argument or ``cluster-id`` ConfigMap option. The value must be between 1 and
 
 Repeat this step for each cluster.
 
+.. note::
+
+   This can also be done by passing `--set global.cluster.id=<id>` and
+   `--set global.cluster.name=<name>` to `helm template` when installing or
```

Comment:
needs to be double ` between `--set global.cluster.name=<name>` and `helm template`